### ISSUE TITLE
Error: Mismatched anonymous define() module: 

### ISSUE THREAD

**Issue by y-a-v-a** [WICG/inert#114]
Hi,
I still use requirejs: https://requirejs.org/ . Salesforce uses your module in their LiveChat. Your built version uses an anonymous define. RequireJS doesn't like that (see also https://requirejs.org/docs/errors.html ). LiveChat breaks our pages due to the anonymous define. This could quite easily be fixed, for example like this:
```javascript
(function (global, factory) {
  typeof exports === 'object' && typeof module !== 'undefined' ? factory() :
  typeof define === 'function' && define.amd ? define(['inert'], factory) : // give it a name
  (factory());
}(this, (function () { 'use strict';
```
Cheers Vincent

**Comment by y-a-v-a** [comment]
Easiest of course would be to update the configuration in rollup.config.js and add a name property for the output. See also https://rollupjs.org/guide/en#big-list-of-options

**Comment by y-a-v-a** [comment]
Fixed in #119
